common/common.go

```diff
@@ -548,7 +548,7 @@ func GetKeyTagMap(src *Tag) *Tag {
 	return res
 }
 
-//Get value tag map for map
+// Get value tag map for map
```

Comment:
These changes came automatically from editor if making any change in file. Keeping them.